### PR TITLE
cm modify file

### PR DESCRIPTION
**Pull request by McDceci**
New file pull request

### CODE REVIEW

**Review by gabeg1163 — COMMENTED** [review]
Modified approved.

File: _slides/05-McDceci.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
```
Comment (gabeg1163) [inline _slides/05-McDceci.md:6]: Gabriel

**Review by weshockley — APPROVED** [review]
good to go!

**Review by dmmartin76 — APPROVED** [review]

**Review by McDceci — COMMENTED** [review]
hello

**Review by McDceci — COMMENTED** [review]
cmd

**Comment by patlamza** [comment]
ok

**Comment by patlamza** [comment]
ok

**Comment by gabeg1163** [comment]
Looks Good

**Pull request closed without merging**